spec/index.html

```diff
@@ -563,7 +569,26 @@ <h2>Example</h2>
                      "value": "&lt;p xmlns=\"http://www.w3.org/1999/xhtml\"&gt;My name is &lt;b&gt;alice&lt;/b&gt;&lt;/p&gt;"
                    },
 
-                   "friend" : { "type": "bnode", "value": "r2" }
+                   "friend" : { "type": "bnode", "value": "r2" },
+
+                   "quoted" : {
```

Comment:
Slotting it into the existing example may not best show that it is a future possibility.

Could this be shown as a separate example, possibly even as a fragment of a JSON document?

e.g. "Quoted triple terms can be incorporated into results as shown in the next example:?
(and that also means the main example agrees with `output.srx`.)

Otherwise, I think we're good to go getting this into the document after the discussion at the telecon of 2023-04-13. The section is already "not normative."